content/release-notes/1.20.0.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2020-10-06
+linktitle: "1.20.0"
+title: "1.20.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Disconnected nodes can be deleted from the cluster management page of the console on embedded clusters.
+* [Collect](https://troubleshoot.sh/docs/collect/), [Analyze](https://troubleshoot.sh/docs/analyze/), and [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) specs now support template functions.
```

Comment:
We should mention somewhere that http endpoint is now gone, and maybe even explicitly say we're using a new in-cluster secret mechanism? Last bit might be TMI.

Reply:
Ah, I see that mentioned below.